tendermint/src/merkle.rs

```diff
@@ -154,7 +97,7 @@ mod tests {
         let empty_tree_root = &hex::decode(empty_tree_root_hex).unwrap();
         let empty_tree: Vec<Vec<u8>> = vec![vec![]; 0];
 
-        let root = simple_hash_from_byte_vectors::<Sha256>(&empty_tree);
+        let root = DefaultMerkleHash.simple_hash_from_byte_vectors(&empty_tree);
```

Comment:
This looks non-idiomatic, the object instance has no purpose here, as explained above.